dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/util/SipLogExtension.java

```diff
@@ -141,12 +140,15 @@ public static void destroy() {
      * @return SAS ID
      */
     private static String getSasId() {
-    	
+    	try {
     	SipApplicationSession sas = ThreadLocalStorage.getApplicationSession();
     	if (sas != null) {
     		return sas.getId(); 
     	}
     	
+    	} catch (NullPointerException e) {
+    		
+    	}
```

Comment:
What are the NPE catches for?

Reply:
the logger code was throwing NPEs before I wrapped that section of code into try / catch

Reply:
Should have replied here instead about adding comments as it's just not clear by the code alone.